apps/content/schemaTypes/documents/hackathon.ts

```diff
@@ -0,0 +1,116 @@
+import {defineField, defineType} from 'sanity'
+import {RocketIcon} from '@sanity/icons'
+
+export const hackathon = defineType({
```

Comment:
I made sure that the schema for this lined up with the metadata you had in the mdx files for the hackathon blog posts